src/Analysis/Engine/Impl/Values/BuiltinInstanceInfo.cs

```diff
@@ -141,7 +139,7 @@ public BuiltinInstanceInfo(BuiltinClassInfo klass)
         public override IEnumerable<OverloadResult> Overloads {
             get {
                 IAnalysisSet callRes;
-                if (_klass.GetAllMembers(ProjectState._defaultContext).TryGetValue("__call__", out callRes)) {
+                if (ClassInfo.GetAllMembers(ProjectState._defaultContext).TryGetValue("__call__", out callRes)) {
```

Comment:
In `TypeId`, `ClassInfo` is checked for `null`, and here it isn't. Which one is correct?

Reply:
I checked ctor calls and I don't see how it can be null. And what would be the legal scenario for the null case...